### PR TITLE
console: Fixes update issue in ConsolePlugin

### PR DESCRIPTION
**Pull request by bipuladh**
ConsolePlugin gets correctly updated in upgrade scenarios

Fixes [Bug 2258015](https://bugzilla.redhat.com/show_bug.cgi?id=2258015)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -105,7 +105,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-build: godeps-update test-setup ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	podman build -t ${IMG} .
```
Comment (iamniting) [inline Makefile:108]: Can you revert this change

**Review by iamniting — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (bipuladh) [reply Makefile:108]: oops. this shouldn't have been part of the PR. changing.

**Review by iamniting — APPROVED** [review]
